### PR TITLE
40768: Add 2FA Procedures

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/40768/add-vendor-content-about-managing-security-for-team-members

### CODE REVIEW

File: docs/vendor/team-management-two-factor-auth.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Managing Two-Factor Authentication
+
+This topic describes how to enable and disable Replicated two-factor authentication for individual and team accounts in the Replicated vendor portal.
+
+Alternatively, you can use Google Authentication or SAML Authentication to access the vendor portal. For more information about those options, see [Managing Google Authentication](team-management-google-auth) and [Managing SAML Authentication](team-management-saml-auth).
+
+## About Two-Factor Authentication
+
+Two-factor authentication (2FA) provides additional security by requiring two methods of authentication to access resources and data. When you enable the 2FA option in the vendor portal, you are asked to provide an authentication code and your password during authentication. Replicated utilizes the open algorithm known as the Time-based One-time Password (TOTP 7), which is specified by the Internet Engineering Task Force (IETF) under RFC 6238 2.
```
Comment (paigecalvert) [inline docs/vendor/team-management-two-factor-auth.md:9]: utilizes > uses

File: docs/vendor/team-management-two-factor-auth.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+:::note
+If SAML Authentication or Google Authentication is configured for your team, then 2FA is bypassed, if it is enabled. You can leave 2FA enabled, but you are not prompted to enter a code when logging in.
+:::
```
Comment (paigecalvert) [inline docs/vendor/team-management-two-factor-auth.md:13]: The wording was a little hard to parse in this note, but it sounds like the headline might be that 2FA is not supported if you enable/configure SAML or Google Auth. Maybe this could be formatted as a limitation?

File: docs/vendor/team-management-two-factor-auth.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+1. In the **Two-Factor Authentication** pane, click **Turn off two-factor authentication**.
+
+    <img src="/images/vendor-portal-2fa-turn-off.png" alt="Turn off 2FA in the vendor portal" width="300"/>
+
+1. In the **Confirm password** dialog, enter your vendor portal account password. Click **Confirm password**.
+
+## Enable or Disable 2FA for a Team
+
+As an administrator, you can enable and disable 2FA for teams. You must first enable 2FA on your individual account before you can enable 2FA for teams. Then team members can enable their individual accounts.
```
Comment (paigecalvert) [inline docs/vendor/team-management-two-factor-auth.md:64]: >Then team members can enable their individual accounts.

Suggestion to clarify: "After you enable 2FA for your team, team members can enable 2FA on their individual accounts."

**Review by paigecalvert — APPROVED** [review]
Very well done! Well described, and the screenshots are helpful. Just a few minor comments.

**Comment by jonquil2002** [comment]
Thanks, @paigecalvert  I made the suggested changes. Please see if you think the Limitations section is easier to parse now.

**Review by paigecalvert — APPROVED** [review]
I like it!
